src/migrate_config_secrets/migrate_config_secrets/migrate.py

```diff
@@ -92,55 +80,22 @@ def migrate_config_secrets(
       configmap_obj, artsy_project, artsy_env
     )
 
-  logging.info('Configuring vars in Vault...')
-  update_vault(vault_client, configmap_obj, sensitive_vars, dry_run)
-
-  if dry_run:
-    logging.info('Skipping the rest because this is a dry run.')
-  else:
-    logging.info('Syncing vars from Vault to k8s secret...')
-    sync_vault_k8s_secret(
-      kctl, vault_client, secret_obj, artsy_project, sensitive_vars
-    )
-
-    logging.info('Comparing k8s secret with configmap...')
-    compare_k8s_secret_configmap(secret_obj, configmap_obj, sensitive_vars)
-
-    logging.info('Deleting vars from configmap...')
-    project_repo_dir = os.path.join(repos_base_dir, artsy_project)
-    env_unset(project_repo_dir, artsy_env, sensitive_vars)
+  if not dry_run:
+    logging.info('Moving vars from ConfigMap to Vault...')
+    move_vars(vault_client, configmap_obj, sensitive_vars, repos_base_dir, artsy_project, artsy_env)
 
-def sync_vault_k8s_secret(
-  kctl,
-  vault_client,
-  secret_obj,
-  artsy_project,
-  sensitive_vars
-):
-  logging.info('Force sync Vault -> k8s secret...')
-  epoch_time = int(time.time())
-  kctl.annotate(
-    'externalsecret', artsy_project, f'force-sync={epoch_time}'
-  )
-  # allow annotate to finish
-  time.sleep(10)
-
-  # confirm the two are in sync, var by var
-  logging.info(
-    'Comparing values in Vault with values in k8s secret...'
-  )
-  for var in sensitive_vars:
-    logging.debug(f'Comparing {var} ...')
-    vault_value = vault_client.get(var)
-    k8s_secret_value = secret_obj.get(var)
-    match_or_raise(
-      vault_value,
-      config_secret_sanitizer(k8s_secret_value)
-    )
-
-def update_vault(vault_client, configmap_obj, var_names, dry_run):
+def move_vars(vault_client, configmap_obj, var_names, repos_base_dir, artsy_project, artsy_env):
   ''' configure vars in Vault '''
+  project_repo_dir = os.path.join(repos_base_dir, artsy_project)
   for var in var_names:
-    logging.debug(f'Updating {var} value in Vault...')
+    logging.info(f'Moving {var} from ConfigMap to Vault...')
     configmap_value = configmap_obj.get(var)
-    vault_client.get_set(var, configmap_value, dry_run)
+    vault_client.get_set(var, configmap_value)
+    logging.info('Checking if Vault and configmap have same value...')
+    compare_vault_configmap(vault_client, configmap_obj, var)
+    logging.info('Same value.')
```

Comment:
looking at the log patterns here I think there'll be a repeated log in debug mode, I think it would be cleaner to just have the one `log.info` call in `match_or_raise`

Reply:
yeah interesting point... I think `info` is good in these cases, as I imagine a lot people using this tool are wondering what it's actually doing and I think the current logs are informative without being too much info

Reply:
done